sudo_plugin/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name    = "sudo_plugin"
-version = "1.2.0"
+version = "1.3.0"
```

Comment:
Mind backing this out? I'll version-bump when I cut a release.

Reply:
Just backed it out, thanks